tools/send-to-caas/send-utils.js

```diff
@@ -253,7 +253,21 @@ const getDateProp = (dateStr, errorMsg) => {
 
 const getOrigin = () => {
   const origin = getConfig().project || getConfig().repo;
-  if (origin) return origin;
+  const mappings = {
+    cc: 'hawks',
+    dc: 'doccloud',
+    bacom: 'bacom',
+    experienceleague: 'experienceleague',
+    magento: 'magento',
+    marketo: 'marketo',
+    milo: 'milo',
+    northstar: 'northstar',
+    workfront: 'workfront',
+  };
+
+  if (mappings[origin]) {
+    return mappings[origin] || origin;
```

Comment:
Nit: Since the default is `origin` we can remove all mappings that have key/values that are identical.